service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionServiceTest.java

```diff
@@ -283,7 +283,9 @@ void shouldGetDefaultPreAttendance() {
             .hearingTime("3:30pm - 4:30pm")
             .hearingAttendance("Remote - video call")
             .preHearingAttendance(DEFAULT_PRE_ATTENDANCE)
-            .hearingVenue("Remote hearing at Venue. Details and instructions will be sent by the local court.")
+            .hearingVenue(
+                String.format("Remote hearing at %s. Details and instructions will be sent by the local court.",
```

Comment:
why not make REMOTE_HEARING_VENUE protected so that it's accessible here

Reply:
changed